conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinDbViewsTest.scala

```diff
@@ -0,0 +1,61 @@
+package tech.cryptonomic.conseil.common.bitcoin
+
+import scala.concurrent.ExecutionContext
+
+import cats.effect._
+import org.scalatest.{Matchers, WordSpec}
+import slick.jdbc.PostgresProfile.api._
+
+import tech.cryptonomic.conseil.common.testkit.InMemoryDatabase
+
+class BitcoinDbViewsTest
+    extends WordSpec
+    with Matchers
+    with InMemoryDatabase
+    with BitcoinInMemoryDatabaseSetup
+    with BitcoinFixtures
+    with BitcoinStubs {
+
+  implicit val contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
+
+  "Bitcoin db views" should {
+
+      "return balance for the given address" in new BitcoinPersistenceStubs(dbHandler) {
+        (for {
+          // run
+          _ <- tx.transact(
+            bitcoinPersistanceStub
+              .createBlock(RpcFixtures.blockResult, List(RpcFixtures.transactionResult))
+          )
+          // create view, TODO: Fix Tables.scala to handle view in a proper way
+          _ <- tx.transact(sqlu"""
+            CREATE OR REPLACE VIEW bitcoin.accounts AS
+              SELECT
+                script_pub_key_addresses AS address,
+                SUM(
+                  CASE WHEN bitcoin.inputs.output_txid IS NULL THEN
+                    value
+                  ELSE
+                    0
+                  END) AS value
+              FROM
+                bitcoin.outputs
+                LEFT JOIN bitcoin.inputs ON bitcoin.outputs.txid = bitcoin.inputs.output_txid
+                  AND bitcoin.outputs.n = bitcoin.inputs.v_out
+                GROUP BY
+                  bitcoin.outputs.script_pub_key_addresses
+          """)
+          // test results
+          _ <- tx.transact(Tables.Blocks.result)
+          _ <- tx.transact(Tables.Transactions.result)
+          _ <- tx.transact(Tables.Inputs.result)
+          _ <- tx.transact(Tables.Outputs.result)
+
```

Comment:
Is it needed to read and ignore all tables results? Will it check something that might go wrong?

Reply:
my mistake